### PR TITLE
Material types in singular

### PR DESCRIPTION
**Pull request by Adamik10**
#### Link to issue
https://reload.atlassian.net/browse/DDFBRA-253

#### Description
This PR:
- shows material types in SlideSelect in singular
- shows material types in Details info box in singular
- shows a dash "-" for empty values for improved UX
- shows contributors in work page header in case work creators aren't defined

#### Screenshot of the result
image
image
image

#### Additional comments or questions
-

### CODE REVIEW

File: components/shared/workCard/helper.ts
Comment (ThomasGross) [inline components/shared/workCard/helper.ts:1]: displayCreators method should be moved to a global helper file now its not only used in workcard

File: components/shared/workCard/helper.ts [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
   GeneralMaterialTypeCodeEnum,
+  ManifestationDetailsFragment,
   SearchWithPaginationQuery,
+  WorkFullWorkPageFragment,
   WorkTeaserSearchPageFragment,
 } from "@/lib/graphql/generated/fbi/graphql"
 
 export const displayCreators = (
-  creators: SearchWithPaginationQuery["search"]["works"][0]["creators"],
+  creators: WorkFullWorkPageFragment["creators"] | ManifestationDetailsFragment["contributors"],
```
Comment (ThomasGross) [inline components/shared/workCard/helper.ts:10]: I'm suggesting not having the function to be specific to the types WorkFullWorkPageFragment["creators"] and  ManifestationDetailsFragment["contributors"]. Could it expect an array instead here?

```suggestion
  creators: string[],
```

**Review by ThomasGross — APPROVED** [review]
Looks good i have a few comments to consider

File: components/shared/workCard/helper.ts (hunk 1 above)
Comment (Adamik10) [reply components/shared/workCard/helper.ts]: That's not specific enough for this case. Then TS wouldn't be happy with us giving it an array of objects. I think it's pretty nice for a specific use case :)

**Pull request merged**